terragrunt/env/production/api/terragrunt.hcl

```diff
@@ -2,6 +2,41 @@ terraform {
   source="git::https://github.com/cds-snc/url-shortener//terragrunt/aws/api?ref=${get_env("INFRASTRUCTURE_VERSION")}"
 }
 
+dependencies {
+  paths = ["../network", "../ecr"]
+}
+
+dependency "network" {
+  config_path = "../network"
+
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs_merge_with_state           = true
+  mock_outputs = {
+    api_security_group_id = ""
+    private_subnet_ids = [""]
+  }
+}
+
+dependency "ecr" {
+  config_path = "../ecr"
+
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs_merge_with_state           = true
+  mock_outputs = {
+    ecr_repository_arn = ""
+    ecr_repository_url = ""
+  }
+}
+
+inputs = {
+  api_security_group_id = dependency.network.outputs.api_security_group_id
+  private_subnet_ids	= dependency.network.outputs.private_subnet_ids
+  ecr_repository_arn	= dependency.ecr.outputs.ecr_repository_arn
+  ecr_repository_url	= dependency.ecr.outputs.ecr_repository_url
+  ecr_tag		= "latest"
+  url_shortener_table_name = "url_shortener_table"
```

Comment:
This is totally fine for now, but once we've got the DynamoDB module finished, we'll change this to a dependency and set the table name from that module's output.

Reply:
You are totally correct - I just put a sample name there since it was not decided what the table will be called, but will definitely change it once we know.